manifests/qna-rag-modules.yaml

```diff
@@ -0,0 +1,23 @@
+name: qna-rag
+path: modules/fmops/qna-rag
+parameters:
+  - name: cognito-pool-id
+    value: us-east-1-XXXXX
```

Comment:
Looks like we need an [IDF](https://github.com/awslabs/idf-modules) module for Cognito. Can you move the manifest in `examples/manifests/` since this requires user input/pre-setup Conginto user pool? We can move it back once we have congnito module/or IAM-only auth mode.

Reply:
Moved the manifest to examples

Reply:
Thanks!